flojoy/flojoy_python.py

```diff
@@ -1,4 +1,6 @@
+from contextlib import ContextDecorator
```

Comment:
Need not be addressed: Should we group imports by the syntax used? If yes, we might want to move this import in the second group.